src/github/copilotRemoteAgent/chatSessionContentBuilder.ts

```diff
@@ -388,6 +388,10 @@ export class ChatSessionContentBuilder {
 			const toolDetails = parseToolCallDetails(toolCall, deltaContent);
 			toolPart.toolName = toolDetails.toolName;
 
+			if (toolPart.toolName === 'think') {
```

Comment:
It looks like this `think` string is used in many places. It would be great to have some string constants for all the tools we know about.